### PR TITLE
Delete repeated title and fix some labels

### PR DESCRIPTION
**Pull request by ahed-compucorp**
Overview
----------------------------------------
This a fellow-up PR after https://github.com/civicrm/civicrm-core/pull/19531 which will :
- Fix some titles and labels that contain the name of its own entity
- Fix repeated title.
- Fix label typos.
- Add label for fields without title.

### CODE REVIEW

File: xml/schema/Bridge/OG.xml [hunk 1]
```diff
@@ -24,6 +25,9 @@
     <name>drupal_og_id</name>
     <type>int unsigned</type>
     <comment>Foreign key to the organic group.</comment>
+    <html>
+      <label>Drupal Organic Group</label>
+    </html>
```
Comment (colemanw) [inline xml/schema/Bridge/OG.xml:30]: This file appears to be unused. I think this is from an old module.
Please remove changes to this file. It's not important.

File: xml/schema/Bridge/Role.xml [hunk 2]
```diff
@@ -24,6 +25,9 @@
     <name>drupal_role_id</name>
     <type>int unsigned</type>
     <comment>Foreign key to the drupal role.</comment>
+    <html>
+      <label>Drupal Role</label>
+    </html>
```
Comment (colemanw) [inline xml/schema/Bridge/Role.xml:30]: This file is also unused. Please remove changes to this file.

File: xml/schema/Contact/Household.xml [hunk 3]
```diff
@@ -47,6 +48,9 @@
     <dataPattern>/^\w+$/</dataPattern>
     <comment>Household Name.</comment>
     <fulltext/>
+    <html>
+      <label>Name</label>
+    </html>
```
Comment (colemanw) [inline xml/schema/Contact/Household.xml:53]: I think this file is obsolete. Please remove these changes.

File: xml/schema/Contact/Individual.xml [hunk 4]
```diff
@@ -193,6 +230,9 @@
     <comment>OPTIONAL FK to civicrm_contact_household record. If NOT NULL, direct mail communications to household
       rather than individual location.
     </comment>
+    <html>
+      <label>Mail to Household</label>
+    </html>
```
Comment (colemanw) [inline xml/schema/Contact/Individual.xml:235]: I think this file is obsolete. Please remove changes to this file.

File: xml/schema/Contact/Organization.xml [hunk 5]
```diff
@@ -71,6 +78,9 @@
     <import>true</import>
     <headerPattern>/^sic|(s(ic\s)?code)$/i</headerPattern>
     <comment>Standard Industry Classification Code.</comment>
+    <html>
+      <label>SIC Code</label>
+    </html>
```
Comment (colemanw) [inline xml/schema/Contact/Organization.xml:83]: I think this file is obsolete. Please remove these changes.

**Comment by colemanw** [comment]
Thanks for this PR. I've confirmed that some xml files are unused and have made #19570 to delete them.

**Comment by ahed-compucorp** [comment]
I removed them from the last two commits.

**Comment by eileenmcnaughton** [comment]
test this please

**Comment by seamuslee001** [comment]
This all looks good to me merging

**Pull request merged**